include/saber/geometry/point.hpp

```diff
@@ -123,6 +126,40 @@ inline constexpr bool Point<T>::IsEqual(const Point& inPoint) const
     return result;
 }
 
+// Structured Binding Support
+
+template<std::size_t Index, typename T>
+inline T get(const Point<T>& inPoint)
+{
+    static_assert(Index < 2, "Unexpected Index for Point<T>");
```

Comment:
hmmm... magic number here: "2"
We want to reduce usage of "magic numbers", because it's hard to track them all down if they ever need to be changed. Consider validating the magic number, so you'll get a compiler error if you forget to change it:
```
    static_assert(2 == std::tuple_size<Point<T>>::value);
    static_assert(Index < 2, "Unexpected Index for Point<T>");
```